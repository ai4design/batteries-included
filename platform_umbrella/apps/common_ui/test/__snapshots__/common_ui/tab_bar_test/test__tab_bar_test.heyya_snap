<div class="pb-8 ">
  <div class="gap-4 lg:gap-6 flex isolate rounded-lg border-gray-200 dark:bg-gray-800 dark:border-gray-600 border flex-col mx-5 lg:flex-row lg:mx-0" aria-label="Tabs">
  
    
  <a href="#" class="group relative min-w-0 flex-1 overflow-hidden py-3 px-4 text-sm font-medium text-center focus:z-10 rounded-lg text-gray-700 hover:bg-gray-100 dark:hover:bg-gray-700 dark:text-gray-300 p-4" phx-click="test">
  
  <span>Title</span>
  <span aria-hidden="true" class="bg-transparent absolute inset-x-0 bottom-0 h-0.5"></span>

</a>
  <a href="/second_path" data-phx-link="patch" data-phx-link-state="push" class="group relative min-w-0 flex-1 overflow-hidden py-3 px-4 text-sm font-medium text-center focus:z-10 rounded-lg text-gray-700 hover:bg-gray-100 dark:hover:bg-gray-700 dark:text-gray-300 p-4">
  
  <span>Another Title</span>
  <span aria-hidden="true" class="bg-transparent absolute inset-x-0 bottom-0 h-0.5"></span>

</a>
  <a href="/last_path" data-phx-link="redirect" data-phx-link-state="push" class="group relative min-w-0 flex-1 overflow-hidden py-3 px-4 text-sm font-medium text-center focus:z-10 rounded-lg text-gray-700 hover:bg-gray-100 dark:hover:bg-gray-700 dark:text-gray-300 p-4">
  
  <span>Yet Another Title</span>
  <span aria-hidden="true" class="bg-transparent absolute inset-x-0 bottom-0 h-0.5"></span>

</a>

  
</div>
</div>