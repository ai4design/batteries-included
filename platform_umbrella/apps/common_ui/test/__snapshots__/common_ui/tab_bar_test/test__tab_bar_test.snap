"<div class=\"block pb-8\">\n  <nav class=\"flex isolate rounded-lg shadow divide-gray-200 flex-col mx-5 lg:flex-row lg:divide-x lg:mx-0\" aria-label=\"Tabs\">\n    <a href=\"/path\" data-phx-link=\"redirect\" data-phx-link-state=\"push\" class=\"text-gray-500 hover:text-gray-700 group relative min-w-0 flex-1 overflow-hidden bg-white py-4 px-4 text-sm font-medium text-center hover:bg-gray-100 focus:z-10\">\n  \n      <span>Title</span>\n      <span aria-hidden=\"true\" class=\"bg-transparent absolute inset-x-0 bottom-0 h-0.5\"></span>\n    \n</a><a href=\"/second_path\" data-phx-link=\"redirect\" data-phx-link-state=\"push\" class=\"text-gray-900 rounded-l-lg group relative min-w-0 flex-1 overflow-hidden bg-white py-4 px-4 text-sm font-medium text-center hover:bg-gray-100 focus:z-10\">\n  \n      <span>Another Title</span>\n      <span aria-hidden=\"true\" class=\"bg-pink-500 absolute inset-x-0 bottom-0 h-0.5\"></span>\n    \n</a><a href=\"/last_path\" data-phx-link=\"redirect\" data-phx-link-state=\"push\" class=\"text-gray-500 hover:text-gray-700 group relative min-w-0 flex-1 overflow-hidden bg-white py-4 px-4 text-sm font-medium text-center hover:bg-gray-100 focus:z-10\">\n  \n      <span>Yet Another Title</span>\n      <span aria-hidden=\"true\" class=\"bg-transparent absolute inset-x-0 bottom-0 h-0.5\"></span>\n    \n</a>\n  </nav>\n</div>"